Old/iR2_Alg MP.jl

```diff
@@ -30,16 +156,16 @@ mutable struct iR2RegParams
   verb::Bool
   activate_mp::Bool
   flags::Vector{Bool}
-  κf::Real
-  κh::Real
-  κ∇::Real
-  κs::Real
-  κξ::Real
-  σk::Real
-  ν::Real
+  κf::T
+  κh::T
+  κ∇::T
+  κs::T
+  κξ::T
+  σk::H # H is the "Highest" floating point format in Π.
```

Comment:
I'm a bit confused, isn't `H` the type of `h`, the argument of `iR2` (i.e. a function)? How can it be the type of sigma k as well ?